release_notes/ocp-4-17-release-notes.adoc

```diff
@@ -59,6 +59,20 @@ With this release, you can now use DNF to install additional packages to your cu
 [id="ocp-4-17-installation-and-update_{context}"]
 === Installation and update
 
+[id="ocp-4-17-installing-a-cluster-with-multiarch-support_{context}"]
+==== Installing a cluster with the support for configuring multi-architecture compute machines
+
+With this release, you can install an {aws-first} cluster and {gcp-first} cluster with the support for configuring multi-architecture compute machines. After installing the cluster, you can add multi-architecture compute machines to the cluster in the following ways:
+
+* Adding 64-bit x86 compute machines to a cluster that uses 64-bit ARM control plane machines and already includes 64-bit ARM compute machines. In this case, 64-bit x86 is considered the secondary architecture.
+* Adding 64-bit ARM compute machines to a cluster that uses 64-bit x86 control plane machines and already includes 64-bit x86 compute machines. In this case, 64-bit ARM is considered the secondary architecture.
+
```

Comment:
In 4.17 we added the ability to deploy an arm64 control planes with x86 workers at install time (and vice versa):
 arm64 cp + x86 workers 
 and 
 x86 cp + arm64 workers.

Reply:
Updated as follows:

* Adding 64-bit x86 compute machines to a cluster that uses 64-bit ARM control plane machines.
* Adding 64-bit ARM compute machines to a cluster that uses 64-bit x86 control plane machines.

Reply:
@sr1kar99 not quite.. we already support adding multi-arch nodes to existing clusters. We now support installing a cluster with a controlplane and workers of different architectures. 
Something like: 
```
With this release, you can install an {aws-first} cluster and {gcp-first} cluster with the support for configuring multi-architecture compute machines. At install time you can specifiy different cpu architecures for the control plane and worker machines  in the following ways:

*  64-bit x86 compute machines and  64-bit ARM control plane machines.
*  64-bit ARM compute machines  and 64-bit x86 control plane machines.```

Reply:
Hi @jeffdyoung 
Got it! I missed this point.
Updated as per the suggestion.